### PR TITLE
Fix some typo and markdown in troubleshooting guide

### PR DESCRIPTION
**Pull request by kirillbilchenko**
Fix some typo's and markdown for troubleshooting document

### CODE REVIEW

**Comment by kirillbilchenko** [comment]
@mumoshu should I close the pr? I mean no one checking it, and I'm not sure if I did something wrong?

**Comment by Link-** [comment]
Let's keep it @kirillbilchenko - we're very low on bandwidth as we push the new mode out, we'll check this as soon as we can.

Thank you for your contribution 🙇

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -1,60 +1,63 @@
...
+    - [InternalError when calling webhook: context deadline exceeded](#internalerror-when-calling-webhook-context-deadline-exceeded)
+    - [Invalid header field value](#invalid-header-field-value)
+    - [Helm chart install failure: certificate signed by unknown authority](#helm-chart-install-failure-certificate-signed-by-unknown-authority)
+  - [Operations](#operations)
+    - [Stuck runner kind or backing pod](#stuck-runner-kind-or-backing-pod)
+    - [Delay in jobs being allocated to runners](#delay-in-jobs-being-allocated-to-runners)
+    - [Runner coming up before network available](#runner-coming-up-before-network-available)
+    - [Outgoing network action hangs indefinitely](#outgoing-network-action-hangs-indefinitely)
+    - [Unable to scale to zero with TotalNumberOfQueuedAndInProgressWorkflowRuns](#unable-to-scale-to-zero-with-totalnumberofqueuedandinprogressworkflowruns)
+    - [Slow / failure to boot dind sidecar (default runner)](#slow--failure-to-boot-dind-sidecar-default-runner)
```
Comment (Link-) [inline TROUBLESHOOTING.md]: I'm not sure what changed here besides adding `Troubleshooting` and updating the hierarchy. I don't see the purpose of having the h1 added to the TOC here.

File: TROUBLESHOOTING.md [hunk 2]
```diff
@@ -1,60 +1,63 @@
...
 
 ## Tools
 
 A list of tools which are helpful for troubleshooting
 
-* https://github.com/rewanthtammana/kubectl-fields Kubernetes resources hierarchy parsing tool
-* https://github.com/stern/stern Multi pod and container log tailing for Kubernetes
+- https://github.com/rewanthtammana/kubectl-fields Kubernetes resources hierarchy parsing tool
+- https://github.com/stern/stern Multi pod and container log tailing for Kubernetes
```
Comment (Link-) [inline TROUBLESHOOTING.md]: What's the purpose of chaging the `-` to `*`?

File: TROUBLESHOOTING.md [hunk 3]
```diff
@@ -1,60 +1,63 @@
...
 
 ## Installation
 
 Troubeshooting runbooks that relate to ARC installation problems
 
 ### InternalError when calling webhook: context deadline exceeded
 
-**Problem**
+****Problem****
```
Comment (Link-) [inline TROUBLESHOOTING.md]: What's the point of this change?

File: TROUBLESHOOTING.md [hunk 4]
```diff
@@ -1,60 +1,63 @@
...
 This issue can come up for various reasons like leftovers from previous installations or not being able to access the K8s service's clusterIP associated with the admission webhook server (of ARC).
 
-```
+```text
 Internal error occurred: failed calling webhook "mutate.runnerdeployment.actions.summerwind.dev":
 Post "https://actions-runner-controller-webhook.actions-runner-system.svc:443/mutate-actions-summerwind-dev-v1alpha1-runnerdeployment?timeout=10s": context deadline exceeded
 ```
 
-**Solution**
+****Solution****
```
Comment (Link-) [inline TROUBLESHOOTING.md]: What's the point of this change?

File: TROUBLESHOOTING.md [hunk 5]
```diff
@@ -104,22 +107,23 @@ To fix this, you may either:
 **Solution**
 
 Your base64'ed PAT token has a new line at the end, it needs to be created without a `\n` added, either:
-* `echo -n $TOKEN | base64`
-* Create the secret as described in the docs using the shell and documented flags
+
+- `echo -n $TOKEN | base64`
+- Create the secret as described in the docs using the shell and documented flags
```
Comment (Link-) [inline TROUBLESHOOTING.md]: I'm fine with the newline but I don't see the value of switching to `*` from `-`

File: TROUBLESHOOTING.md [hunk 6]
```diff
@@ -292,12 +296,12 @@ You very likely have some dangling workflow jobs stuck in `queued` or `in_progre
...
 
 **Solution**
 
-The solution is to switch to using faster storage, if you are experiencing this issue you are probably using hdd, switch to ssh fixed the problem in my case. Most cloud providers have a list of storage options to use just pick something faster that your current disk, for on prem clusters you will need to invest in some ssds.
+The solution is to switch to using faster storage, if you are experiencing this issue you are probably using hdd, switch to ssd fixed the problem in my case. Most cloud providers have a list of storage options to use just pick something faster that your current disk, for on prem clusters you will need to invest in some ssds.
```
Comment (Link-) [inline TROUBLESHOOTING.md]: These have been addressed in: https://github.com/actions/actions-runner-controller/pull/2116

To avoid conflicts, I say revert these changes

**Review by Link- — CHANGES_REQUESTED** [review]
Thank you 
@kirillbilchenko for improving our docs. I requested some changes before I can merge this.

File: TROUBLESHOOTING.md (hunk 2 above)
Comment (kirillbilchenko) [reply TROUBLESHOOTING.md]: in the end no difference both are valid let me put back

File: TROUBLESHOOTING.md (hunk 3 above)
Comment (kirillbilchenko) [reply TROUBLESHOOTING.md]: this was automatically changed by my editor sorry I missed it, revered back

**Review by Link- — APPROVED** [review]
Thank you
